### DISCUSSION TITLE
making nested runs in ww3

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#972] [General]
I had tried to make a nest run for ww3 6.07 through the following process:

running the global run using ww3_grid.inp and ww3_grid.nml and ww3_prep.inp and ww3_shel.inp.   This runs successfully and I have nest files generated.  Now when I copy the nest1.ww3 to nest for a medplus run (Mediterranean region) to nest.ww3 and run the ww3_grid for the medplus nest using the ww3_grid.inp and ww3_grid.nml for that nest I get the following error:

ERROR: READ_RECT_NML: namelist read error
ERROR: Cannot match namelist object name .
What is going here?  What needs to be tweaked so that I have a successful run of the medplus nest?  Attached are the ww3_grid.inp ww3_grid.nml for both the global and medplus nest along with the namelist.nml file a wind file and ww3_prep.inp and ww3_shel.inp to reproduce this error if need be.  Please help me fix this issue.
[ww3_shel.inp.txt](https://github.com/NOAA-EMC/WW3/files/11131275/ww3_shel.inp.txt)
[ww3_prep.inp.txt](https://github.com/NOAA-EMC/WW3/files/11131257/ww3_prep.inp.txt)
[ww3_grid_medplus.nml.txt](https://github.com/NOAA-EMC/WW3/files/11131260/ww3_grid_medplus.nml.txt)
[ww3_grid_medplus.inp.txt](https://github.com/NOAA-EMC/WW3/files/11131261/ww3_grid_medplus.inp.txt)
[namelists.nml.txt](https://github.com/NOAA-EMC/WW3/files/11131298/namelists.nml.txt)
[ww3_grid_global.nml.txt](https://github.com/NOAA-EMC/WW3/files/11131299/ww3_grid_global.nml.txt)
[ww3_grid_global.inp.txt](https://github.com/NOAA-EMC/WW3/files/11131300/ww3_grid_global.inp.txt)

**Comment by CarstenHansen** [comment]
When the namelists input file ww3_grid_medplus.nml is present, the ww3_grid_medplus.inp is not used.
There is no slash (/) ending the namelist &RECT_NML. I guess this could cause such an error message.

**Reply by JMSnyder1980** [reply]
Yes I did try to put a / in &RECT_NML but still have the same error message.  What does it mean you do not use ww3_grid.inp?

**Reply by CarstenHansen** [reply]
Hi.

The manual for version 6.07, chapter 4.4.2, says: "If inp and nml files are both present in the run directory, the priority will be given to the namelist file."

 I have tested the file ww3_grid_medplus.nml, and it turns out that my version of ww3_grid (compiled with ifort/16.0.3/ieee) doesn't make any complaint about the missing '/'. However, the extra '.' at the end of the line 71: `  RECT%X0                = -15.250.` causes an error:
`ERROR: READ_RECT_NML: namelist read error`
`ERROR: syntax error in NAMELIST input, unit 10, file `(my setup dir)`/./ww3_grid.nml, line 72, position 0
`
 I think the second ERROR line depends on the compiler.

**Reply by JMSnyder1980** [reply]
that fixed the issue. Thank you.
